### PR TITLE
Static player positions, better table scaling

### PR DESCRIPTION
**Pull request by stevenvergenz**

### CODE REVIEW

File: client/util.js [hunk 1]
```diff
@@ -55,8 +55,9 @@ window.AudioContext = window.AudioContext || window.webkitAudioContext;
 			});
 
 			// Set the scale for OBJ models based on the known width of the table model.
-			var tableModelWidth = 297.444;
+			var tableModelWidth = 297.444, tableModelHeight = 103;
 			var objScale = 1 / tableModelWidth * tableRadius * 2;
+			var objVScale = 0.8 / tableModelHeight;
```
Comment (brianpeiris) [inline client/util.js]: Maybe name this `tableRelativeVerticalScale` or something. `objVScale` is not very descriptive.

File: client/util.js [hunk 2]
```diff
@@ -127,11 +128,10 @@ window.AudioContext = window.AudioContext || window.webkitAudioContext;
 				'/static/models/table/HaH_TableMain.obj',
 				'/static/models/table/HaH_TableMain.mtl',
 				function (obj) {
-					obj.rotation.x = Math.PI / 2;
-					obj.scale.multiplyScalar(objScale);
-					var modelHeight = 103;
 					// The surface of the table should always be at 80cm above the ground;
```
Comment (brianpeiris) [inline client/util.js:131]: Remove this if it doesn't make sense

File: client/util.js [hunk 3]
```diff
@@ -127,11 +128,10 @@ window.AudioContext = window.AudioContext || window.webkitAudioContext;
...
-					obj.position.z = -modelHeight * objScale + 0.8;
+					obj.rotation.x = Math.PI / 2;
+					obj.scale.set(objScale, objVScale, objScale);
+					obj.position.setZ(0);
```
Comment (brianpeiris) [inline client/util.js]: Setting it to 0 seems redundant.

File: server/players.js [hunk 4]
```diff
@@ -118,9 +118,24 @@ function join(id, displayName)
...
+	console.log('seating player at', player.seatNum);
+
 	// add player to the end of the turn order
 	game.turnOrder.push(player);
 
+	// push back czar index if joiner is before them
+	if(placeInTurnOrder <= game.czar){
+		game.czar = (game.czar+1) % game.turnOrder.length;
+		console.log('incrementing czar to', game.czar);
+	}
```
Comment (brianpeiris) [inline server/players.js:137]: Hmm, this turnOrder juggling is confusing to me, so maybe I can't really comment on it, but shouldn't the new `game.czar` state be pushed to the clients somehow?

Comment (stevenvergenz) [reply server/players.js:137]: The czar user id is pushed to the clients when each round starts. This is just to make sure that the same person is the czar before and after a new player joins.

**Pull request merged**